test_files/es6/enum.js

```diff
@@ -9,7 +9,6 @@ EnumTest1.XYZ = 0;
 /** @type {EnumTest1} */
 EnumTest1.PI = 3.14159;
 /** @typedef {number} */
-// This additional exported enum is here to exercise the fix for issue #51.
```

Comment:
OOC, what did this do before your fix? Closure didn't take the type? This might be good example of why custom syntax for type declarations is preferable over comments.